schemas/HPXMLDataTypes.xsd

```diff
@@ -309,6 +309,14 @@
 			<xs:enumeration value="attached on three sides"/>
 		</xs:restriction>
 	</xs:simpleType>
+	<xs:simpleType name="VerticalSurroundings">
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="unit above"/>
+			<xs:enumeration value="unit below"/>
+			<xs:enumeration value="unit above and below"/>
+			<xs:enumeration value="no units above or below"/>
```

Comment:
I don't know if we can really improve on the name given that we have
SiteSurroundings describing the same thing horizontally.  That was the time
to pick a better name.  This looks good to me.

On Mon, Nov 30, 2015 at 5:45 PM, Noel Merket <notifications@github.com>
wrote:

> In schemas/HPXMLDataTypes.xsd
> <https://github.com/hpxmlwg/hpxml/pull/64#discussion_r46215829>:
>
> > @@ -309,6 +309,14 @@
> >  			<xs:enumeration value="attached on three sides"/>
> >  		</xs:restriction>
> >  	</xs:simpleType>
> > +	<xs:simpleType name="VerticalSurroundings">
> > +		<xs:restriction base="xs:string">
> > +			<xs:enumeration value="unit above"/>
> > +			<xs:enumeration value="unit below"/>
> > +			<xs:enumeration value="unit above and below"/>
> > +			<xs:enumeration value="no units above or below"/>
>
> Here are the added enumerations.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hpxmlwg/hpxml/pull/64/files#r46215829>.
>